backend/src/main/java/gov/cdc/usds/simplereport/config/simplereport/DemoUserConfiguration.java

```diff
@@ -23,15 +24,19 @@
   private DemoUser defaultUser;
   private List<DemoUser> users;
   private Map<String, DemoUser> byUsername;
+  private Set<String> siteAdminEmails;
 
   public DemoUserConfiguration(List<DemoUser> allUsers) {
-    this(null, allUsers);
+    this(null, allUsers, new ArrayList<>());
```

Comment:
i know that this constructor is only used for testing purposes, but should we outfit it to take a list of siteAdminEmails as a parameter too?

Reply:
I'd say we should wait until we need it.  Fixed the unnecessary empty array here, though.